### DISCUSSION TITLE
Implementing qtoolbox with QSkinny controls

### DISCUSSION THREAD

**Discussion by hestrro** [uwerat/qskinny#490] [Q&A] [answered]
The effect of qtoolbox should be achieved by inserting a box using QskLinerBox's insertitem (index, qquickitem), right? QskLinerBox will automatically handle the position relationship after insertion

**Comment by uwerat** [comment, marked as answer]
No QskLinearBox is the counter part of QBoxLayout.

The functionality of QToolBox is more or less covered by QskSegmentedBar. However you need to customize its skin hints ( actually clearing most of them ).
